kyuubi-common/src/main/scala/org/apache/kyuubi/session/Session.scala

```diff
@@ -29,6 +30,8 @@ trait Session {
 
   def conf: Map[String, String]
 
+  def sessionKyuubiConf: KyuubiConf
```

Comment:
> The name looks too long, what about renaming `conf` to `confOverlay`, and naming it `conf`? 

I see that there are several configurations in the session, `conf`, `sessionConf`, `normalizedConf`.

Reply:
The session conf pipeline is:
```
user request conf(conf in Session) -> normalized conf -> advisor conf
```

Some session implementation have already merged the session conf to KyuubiConf. e.g.  `KyuubiSessionImpl`, so it's not correct to merge normalized conf to kyuubi conf.

I'd like to only do fix the trino and hive operation manager that use normalized conf instead of global kyuubi conf.